packages/@markuplint/config-presets/build.mjs

```diff
@@ -23,7 +23,7 @@ const extended = {};
 
 for (const file of files) {
 	const filename = path.basename(file);
-	const code = await readFile(path.resolve(dirname, 'src', filename), { encoding: 'utf-8' });
+	const code = await readFile(path.resolve(srcDir, filename), { encoding: 'utf-8' });
```

Comment:
@kagankan 

I changed to use `dirname` because I ran this file from the project root directory through the command `yarn site:dev`.
Can you check whether ran correctly from the directory and through the command?

Reply:
@kagankan 

Sorry, it's a misunderstanding.